frontend/pages/policies/ManagePoliciesPage/components/PoliciesListWrapper/PoliciesListWrapper.tsx

```diff
@@ -1,14 +1,24 @@
 import React from "react";
 import { noop } from "lodash";
+import paths from "router/paths";
 
 import Button from "components/buttons/Button";
 import { IPolicy } from "interfaces/policy";
+import { ITeam } from "interfaces/team";
 import TableContainer from "components/TableContainer";
 import { generateTableHeaders, generateDataSet } from "./PoliciesTableConfig";
+// @ts-ignore
+import policySvg from "../../../../../../assets/images/no-policy-323x138@2x.png";
 
 const baseClass = "policies-list-wrapper";
 const noPoliciesClass = "no-policies";
 
+const TAGGED_TEMPLATES = {
+  hostsByTeamPRoute: (teamId: number | undefined | null) => {
+    return `${teamId ? `/&team_id=${teamId}` : ""}`;
```

Comment:
This link is not currently working because it is the first URL param, so needs to start with `?`: 

```
return `${teamId ? `/?team_id=${teamId}` : ""}`;
```